common/src/main/java/com/octopus/teamcity/opentelemetry/common/PluginConstants.java

```diff
@@ -21,7 +21,7 @@ private PluginConstants() {}
     public static final String ATTRIBUTE_SUCCESS_STATUS = TRACER_INSTRUMENTATION_NAME + ".success_status";
     public static final String ATTRIBUTE_FAILED_TEST_COUNT = TRACER_INSTRUMENTATION_NAME + ".failed_test_count";
     public static final String ATTRIBUTE_BUILD_PROBLEMS_COUNT = TRACER_INSTRUMENTATION_NAME + ".build_problems_count";
-    public static final String ATTRIBUTE_ARTIFACT_SIZE = TRACER_INSTRUMENTATION_NAME + "_artifact_size";
+    public static final String ATTRIBUTE_ARTIFACT_SIZE = "_artifact_size";
```

Comment:
I don't have full context on how this is used, but all the others seem to start with `.` for the final suffix. Should this be:
```suggestion
    public static final String ATTRIBUTE_ARTIFACT_SIZE = ".artifact_size";
```

Reply:
Updated! Thanks for the suggestion!